verify.go

```diff
@@ -91,3 +96,31 @@ func (v *Verifier[CustomClaims]) Verify(ctx context.Context, token string) (*cle
 
 	return claims, claims.Custom.(*CustomClaims), nil
 }
+
+func (v *Verifier[CustomClaims]) getJSONWebKey(ctx context.Context, keyID string) (*clerk.JSONWebKey, error) {
```

Comment:
```suggestion
func (v *Verifier[CustomClaims]) fetchJSONWebKey(ctx context.Context, keyID string) (*clerk.JSONWebKey, error) {
```

Que los de Clerk no han sido muy idiomáticos aquí.